cognite_toolkit/cdf_tk/load.py

```diff
@@ -125,13 +125,15 @@ class Loader(ABC, Generic[T_ID, T_Resource, T_ResourceList]):
 
     support_drop = True
     support_upsert = False
+    mode: Literal["load", "all_only"] = "load"
```

Comment:
Why do you add `mode` to the interface? It is only used by the `AuthLoader` so it should be removed from here.